### PR TITLE
DOP-4345 add docs-cpp makefile

### PR DESCRIPTION
**Pull request by anabellabuckvar**
Makefile for new repo

https://github.com/mongodb/docs-cpp

### CODE REVIEW

File: makefiles/Makefile.docs-cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+REPO_DIR=$(shell pwd)
+
+SNOOTY_DB_USR = $(shell printenv MONGO_ATLAS_USERNAME)
+SNOOTY_DB_PWD = $(shell printenv MONGO_ATLAS_PASSWORD)
+
+include ~/shared.mk
+
+
+get-build-dependencies: 
+	@curl https://raw.githubusercontent.com/mongodb/docs-worker-pool/meta/publishedbranches/docs-mongodb-shell.yaml > ${REPO_DIR}/published-branches.yaml
```
Comment (mmeigs) [inline makefiles/Makefile.docs-cpp:21]: You don't need this anymore, yay!! However, for now the get-build-dependencies command needs to be here and cannot be empty. Look at newer repos's Makefiles to see how we've added echo statements instead!

**Comment by mmeigs** [comment]
Left one comment!

File: makefiles/Makefile.docs-cpp (hunk 1 above)
Comment (anabellabuckvar) [reply makefiles/Makefile.docs-cpp]: updated it with the echo statement many of the makefiles have. Is that what you had in mind?

**Review by mmeigs — APPROVED** [review]
Yep! Looks good!

**Pull request merged**